scripts/protocgen.sh

```diff
@@ -35,3 +35,6 @@ rm -rf github.com cosmossdk.io
 go mod tidy
 
 ./scripts/protocgen-pulsar.sh
+
+echo "Formatting protobuf files"
```

Comment:
Seems like this should be at the beginning of the file, like https://github.com/cosmos/cosmos-sdk/pull/17447/commits/77f1ef53c5ce25e52ddfe3dd1af287f880555e8e.
I had to run it twice in #17447 otherwise.